ecocodephp.properties

```diff
@@ -13,8 +13,14 @@ defaults.mavenArtifactId=ecocode-php-plugin
 1.2.1.downloadUrl=https://github.com/green-code-initiative/ecoCode/releases/download/1.2.1/ecocode-php-plugin-1.2.1.jar
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
-1.3.0.description=Updates rules tags�; adds rule (EC3)�; upgrades librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
-1.3.0.sqVersions=[9.9,LATEST]
+1.3.0.description=Updates rules tags; adds rule (EC3); upgrades librairies to SonarQube 10.0.0; clean-up plugins and dependencies
+1.3.0.sqVersions=[9.9,10.1]
```

Comment:
Oops. This one really does need to end with `10.3`
The version being archived - the one that previously ended with `LATEST` needs to be updated to the current version.

Reply:
sorry, correction done.